src/lib/lsStore.js

```diff
@@ -181,7 +183,9 @@ export default function storeData () {
         let storeNow = false
         if (mutation.type === 'change_storage_method') {
           if (!mutation.payload) {
-            useStorage = sessionStorage.setItem('adm-persist', JSON.stringify(state))
+            const clonedState = cloneDeep(state)
+            delete clonedState.areChatsLoading
+            useStorage = sessionStorage.setItem('adm-persist', JSON.stringify(clonedState))
```

Comment:
Did you just add a whole `lodash` as a dependency just to remove a property from the persisted state? If yes, please, undo this. Instead just use:
```sessionStorage.setItem('adm-persist', JSON.stringify(state, (k, v) => (k === 'areChatsLoading' ? undefined : v)))```